changes.md

```diff
@@ -2,6 +2,7 @@
 
 ## Passes
 * `LikeReplacement` pass must be run by all transformers.
+* `ngraph::pass::FusionType` is now a enum class. Constant values defined by `FusionType` are created for backward compatibility and will be removed in future releases.
```

Comment:
Probably should be "an enum class", but up to you if you want to change or not.

Reply:
Thanks, fixed.